Warning! here be dragons.

For versions of Xen before 4.8, the base repository does not have
.gitarchive-info; Fake a suitable one up.  This must be kept in sync
manually.

Because planex processes the patchqueue when creating the srpm, a regular
.gitarchive-info in the patchqueue tarball isn't available to the build
environment.  Fake it up in a patch file to appear as if it were created in
the base tarball.

diff --git a/.gitarchive-info b/.gitarchive-info
new file mode 100644
index 0000000..5be2a1c
--- /dev/null
+++ b/.gitarchive-info
@@ -0,0 +1,2 @@
+Changeset: bcc9e245aafbdae44c761053c898bedb3582cc4d
+Commit date: Mon, 20 Nov 2017 16:01:11 +0100
diff --git a/.gitarchive-info-pq b/.gitarchive-info-pq
new file mode 100644
index 0000000..83e5b86
--- /dev/null
+++ b/.gitarchive-info-pq
@@ -0,0 +1,2 @@
+Changeset: $Format:%H$
+Commit date: $Format:%cD$
